rotkehlchen/errors/misc.py

```diff
@@ -80,3 +80,11 @@ class DBSchemaError(Exception):
 
 class GreenletKilledError(Exception):
     """Raised when a greenlet is killed"""
+
+
+class AccountingError(Exception):
```

Comment:
Name is super generic ... but I don't think we have any other right now. So perhaps fine to leave as is. It's a fatal error in accounting processing. In the future we may have other errors in other parts of accounting, or non fatal ones. 

But guess can easily modify stuff if needed.